docs/resiliency/index.asciidoc

```diff
@@ -56,6 +56,14 @@ If you encounter an issue, https://github.com/elastic/elasticsearch/issues[pleas
 We are committed to tracking down and fixing all the issues that are posted.
 
 [float]
+==== Jepsen Tests
+
+The Jepsen platform is specifically designed to test distributed systems. It is not a single test and is regularly adapted
+to create new scenarios. We have taken over the scenarios tested so far and added them to our testing infrastructure. Of course
```

Comment:
Maybe replace "We have taken over the scenarios tested so far and added them to our testing infrastructure." with "We have adapted all published scenarios to our testing infrastructure."?